src/audio/pipeline/pipeline-schedule.c

```diff
@@ -298,9 +298,9 @@ int pipeline_comp_task_init(struct pipeline *p)
 	return 0;
```

Comment:
@lyakh i think we should enhance the documentation about scheduling and how copy function works across pipeline components.
I was under the impression that with a normal Host -> Volume -> DAI pipeline the copy() function for Host() always got called first.

Reply:
@dbaluta yes, you're right, this is not the final version yet (as the "draft" status is also saying ;-) ), please wait, I think I have a better one now, I'll clean it up and push tomorrow.